DS3231.cpp

```diff
@@ -180,6 +181,57 @@ DateTime RTClib::now(TwoWire & _Wire) {
   return DateTime (y, m, d, hh, mm, ss);
 }
 
+//Subtraction returns difference in seconds between 2 DateTimes.
```

Comment:
In general you can think about to use the function description like this:

```
/**
 * @brief description of function
 * 
 * @param parameter 1 description
 * @param parameter 2 description
 * @return description of return value
 */
```
Then you can hover over the function and it will display the description.